features/checkout/paying_for_order/cancelling_payment_requests_when_payment_method_is_changed.feature

```diff
@@ -0,0 +1,21 @@
+@paying_for_order
+Feature: Cancelling payment requests when payment method is changed
```

Comment:
Just in case,
`Feature` speaks about cancelling multiple requests, but `scenario` cancels only one payment request.